aries/messages/Cargo.toml

```diff
@@ -9,18 +9,23 @@ license.workspace = true
 doctest = false
 
 [dependencies]
-serde = { version = "1.0.167", features = ["derive"] }
-chrono = { version = "0.4.23", features = ["serde"] }
-lazy_static = "1.3"
-serde_json = "1.0.100"
-url = { version = "2.3", features = ["serde"] }
-strum = "0.26.3"
-strum_macros = "0.26.4"
-thiserror = "1.0.37"
-derive_more = "0.99.17"
+serde = { workspace = true, features = ["derive"] }
+serde_json.workspace = true
+chrono = { workspace = true, features = ["serde"] }
+lazy_static.workspace = true
+url = { workspace = true, features = ["serde"] }
+strum.workspace = true
+strum_macros.workspace = true
+thiserror.workspace = true
+derive_more = { workspace = true, features = [
+    "from",
+    "try_into",
+    "display",
+    "error",
+] }
 transitive = "1.0.1"
 isolang = { version = "2.2.0" }
```

Comment:
Missed as well? Unless there's a methodology you were using as to why some things shouldn't be included